app/src/main/java/edu/mit/mitmobile2/tour/adapters/TourStopRecyclerViewAdapter.java

```diff
@@ -25,13 +25,15 @@
     private Context context;
     private LayoutInflater listContainer;
     private TourSelfGuidedCallback callback;
-    private boolean isMainLoop;
+    private boolean isMainLoopRecyclerView;
+    private MITTourStop currentStop;
 
-    public TourStopRecyclerViewAdapter(Context context, List<MITTourStop> tourStops, TourSelfGuidedCallback callback, boolean isMainLoop) {
+    public TourStopRecyclerViewAdapter(Context context, List<MITTourStop> tourStops, MITTourStop currentStop, boolean isMainLoopRecyclerView, TourSelfGuidedCallback callback) {
```

Comment:
Might make more sense just to pass the stop's ID in, since that's the only item in the MITTourStop object we're looking at